release.config.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
     branches: [
         {
             name: "main",
-            level: "minor"
+            level: "none",
+            prerelease: true
```

Comment:
do we need to specify the `channel`?
otherwise it might default to `main` and start tagging things as `1.0.0-main15, -main16...`

or will it remember to use the word `dev` for the prerelease strings?

Reply:
This shouldn't matter because currently Octorelease isn't smart enough to handle versioning for Python projects, so we do it manually and pass the new version to Octorelease. The `level` and `prerelease` properties are still useful for Octorelease to validate the semver diff level and create the GitHub release with prerelease status.

Reply:
Thanks for explaining. I guess that if Octorelease was able to handle versioning for Python projects, then we would've needed to specify which prerelease string to use (`dev`) somehow 😋